Description: Open text files in utf-8 mode, since NEWS.rst contains
 non-ASCII characters.
Author: Barry Warsaw <barry@debian.org>
Bug: https://bugs.launchpad.net/gtimelog/+bug/1263772

--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import os
 import re
 from setuptools import setup
@@ -7,7 +7,7 @@
 
 
 def read(filename):
-    with open(os.path.join(here, filename)) as f:
+    with open(os.path.join(here, filename), 'r', encoding='utf-8') as f:
         return f.read()
 
 
